README.md

```diff
@@ -1,161 +1,38 @@
 [![OpenCollective](https://opencollective.com/mvvmcross/backers/badge.svg)](#backers) 
 [![OpenCollective](https://opencollective.com/mvvmcross/sponsors/badge.svg)](#sponsors)
 [![StackExchange](https://img.shields.io/stackexchange/stackoverflow/t/mvvmcross.svg)](http://stackoverflow.com/questions/tagged/mvvmcross)
 
-MvvmCross is a cross-platform mvvm framework that enables developers to create cross platform apps. 
-
-## Documentation & blog
-
-* See our [official website](http://mvvmcross.com/documentation) for an extensive set of documentation!
-* To stay up-to-date with the latest news go to the [MvvmCross blog](http://mvvmcross.com/blog)!
-
-### Get started
-
-A quick way to get started is to use the Visual Studio MvvmCross [Ninja Coder](https://github.com/asudbury/NinjaCoderForMvvmCross) Extension - download it from [Visual Studio Gallery](https://visualstudiogallery.msdn.microsoft.com/618b51f0-6de8-4f85-95ce-a50c658c7767)
-
-For manual project creation install the necessary packages from [Nuget](https://www.nuget.org/packages?q=mvvmcross)
-
-### Questions & support
-
-* [StackOverflow](http://stackoverflow.com/questions/tagged/mvvmcross)
-* [Xamarin forums](http://forums.xamarin.com)
-* [Slack](https://xamarinchat.herokuapp.com/) join the #mvvmcross channel after you are in
-
-### Documentation and Examples
-
-**The [MvvmCross-Samples](https://github.com/MvvmCross/MvvmCross-Samples) repo contains the latest samples. See the [MvvmCross Wiki](https://github.com/MvvmCross/MvvmCross/wiki) for additional articles and information.**
+👀 Check out [mvvmcross.com](https://www.mvvmcross.com) to get started with MvvmCross 👀
 
-**Samples and Videos:**
+MvvmCross is a cross-platform MVVM framework. It enables developers to create apps using the MVVM pattern on Xamarin and Windows (UWP & WPF). This allows for better code sharing by allowing you to share behavior and business logic between platforms.
 
-* **[N Plus 1 Videos Of MvvmCross][n1videos]**
-* [MvvmCross Tutorials][mvxtuts]
-* [Evolve 2013 talk](http://xamarin.com/evolve/2013#session-dnoeeoarfj)
-* [NDC 2013 talk](http://slodge.blogspot.co.uk/2013/06/ndc2013-mobile-mvvm-and-xamarin-talks.html)
-* [Xaminar - 2012](http://slodge.blogspot.com/2012/12/mvvmcross-video-presentation-xaminar.html)
-* [MvvmCross and Azure - 2012](http://slodge.blogspot.co.uk/2012/11/azure-to-wpmonodroidmonotouch-video-of.html)
+Among the features MvvmCross provides are:
 
+- ViewModel to View bindings using own customizable binding engine, which allows you to create own binding definitions for own custom views
+- ViewModel to ViewModel navigation, helps you share behavior on how and when to navigate
+- Inversion of Control through Dependency Injection and Property Injection
+- Plugin framework, which lets you plug-in cool stuff like GPS Location, Localization, Sensors, Binding Extensions and a huge selection of 3rd party community plug-ins
```

Comment:
MVVM architecture is already implied by the previous text.

Complete flexibility is already mentioned in the paragraph just be low the list.

We could add Platform specifics. I don't think Unit Test helpers is the real selling point here though.